base/interactiveutil.jl

```diff
@@ -64,7 +64,7 @@ function edit(path::AbstractString, line::Integer=0)
     nothing
 end
 
-function edit(m::Method)
+function edit(m::TypeMapEntry)
```

Comment:
Ok, let's do that then. The signature is more or less the most important piece of information about a Method, so it should be easy to see (I believe an example use case just arose in #11242).